### ISSUE TITLE
[SUGGESTION] Add last modified dates to pages

### ISSUE THREAD

**Issue by atopal** [protocol/nft-website#91]
People prefer fresh content and generally want to know how old/new something is. We should add a last modified date to our articles.

Something like: Last modified: May 26, 2021, by NFTschool contributors.

**Comment by jessicaschilling** [comment]
Oddly, these appear on a local build but not when deployed:

![image](https://user-images.githubusercontent.com/1507828/121917148-0c925280-ccf2-11eb-8fcc-91d6580c4600.png)

@zebateira, are you able to look into this? 🙏

**Comment by zebateira** [comment]
It's fixed now.

For reference here's the technical explanation why this was happening:

This issue happened as a combination of two things: error from vuepress code and not using an appropriate fleek docker image.

The "Last Updated" timestamp should be added automatically from the built-in vuepress plugin [`vuepress-plugin-last-updated`](https://vuepress.vuejs.org/plugin/official/plugin-last-updated.html#vuepress-plugin-last-updated), but looking at its code, it seems that they are not catching errors: https://github.com/vuejs/vuepress/blob/369c31520b48131d1c8dcff6dcc95f2699ed68bb/packages/%40vuepress/plugin-last-updated/index.js#L30
So if anything wrong happens, no error is thrown, **the plugin fails silently and the last updated timestamp is not added, which causes the UI to not include the timestamp**.

As @jessicaschilling pointed out, it is showing up in her local machine, so there could be something wrong with fleek's runtime. I debugged the plugin on a preview (#92) and this happened: https://app.fleek.co/#/sites/nft-school/deploys/2021-06-14T16:46:10.826Z?accountId=protocollabs-team

```
5:47:10 PM 06/14/2021: error: Error: spawnSync git ENOENT
```

So the build runtime on fleek did not have git installed because it was using the docker image `node:slim`. I changed it to use `node:lts` which includes git and everything worked.

I'll create an issue or PR to the vuepress repo to address this issue since it should be fixed on their side to throw the error.
